charts/cray-etcd-operator/templates/wait-jobs.yaml

```diff
@@ -27,4 +27,4 @@ spec:
             - '/bin/sh'
           args:
             - '-c'
-            - 'kubectl rollout status deployment -n operators cray-etcd-operator-etcd-operator-etcd-backup-operator cray-etcd-operator-etcd-operator-etcd-operator cray-etcd-operator-etcd-operator-etcd-restore-operator --timeout=15m'
+            - 'kubectl rollout status deployment -n operators cray-etcd-operator-etcd-operator-etcd-backup-operator --timeout=15m; kubectl rollout status deployment -n operators cray-etcd-operator-etcd-operator-etcd-operator --timeout=15m; kubectl rollout status deployment -n operators  cray-etcd-operator-etcd-operator-etcd-restore-operator --timeout=15m'
```

Comment:
Isn't a 45 minute timeout a lot longer than Helm / Loftsman gives the hook to run? I thought the timeout was 5 or 10 mins.

Reply:
probably -- in my testing these come up within seconds, probably don't even need the explicit timeouts.  let me test with that...

Reply:
yeah -- eyeball test is all three deployments were rolled out in about 9 seconds.  removed the explicit timeouts...